WooCommerce/WooCommerceTests/Extensions/IconsTests.swift

```diff
@@ -240,6 +240,10 @@ final class IconsTests: XCTestCase {
         XCTAssertNotNil(UIImage.storeImage)
     }
 
+    func testLaunchStoreImageIsNotNil() {
```

Comment:
super nit: maybe we can follow the snake case as the latest naming style?

Reply:
Done - https://github.com/woocommerce/woocommerce-ios/pull/8948/commits/6ff2dca545b7c77567db9d5f3b38d13ea2ab0e0d